.changeset/young-kiwis-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
```

Comment:
the PR title says `feat` which means it is a new feature, meaning the changeset should be a `minor` but instead it is a `patch`.. is this a feature or a fix?